src/tests/ftest/io/eight_servers_mpiio.py

```diff
@@ -41,7 +41,7 @@ def test_ssf(self):
             transfersize and block size of 32M for 1K transfer size and 128M
             for rest.
 
-        :avocado: tags=ior,mpiio,eightservers,ior_ssf
+        :avocado: tags=ior_ssf
```

Comment:
Does this need the `all` tag?

Reply:
Still wondering about this.

Reply:
I think Scott did explained why we are not putting all tag in this one yet. We can look at it when we are actually running these tests. I would still vote against it though. This is a perf test and should not be included in all tag even if we want to run all the tests available in single go. There should be some distinction.